SU-SWS/stanford_profile
# D8CORE-1291: IE Fixes.

# READY FOR REVIEW 

# Summary
- Fixes alignment and flexbox issues in IE 11

# Needed By (Date)
- Today

# Urgency
- High

# Steps to Test

1. Check out this branch and D8CORE-1291 of jumpstart_ui
2. Clear all caches
3. Open up an IE11 browser window
4. Navigate to the homepage and a few interior pages.
5. Resize broswer window and ensure alignment and spacing.

# Affected Projects or Products
- D8CORE 
- IE Users

# Associated Issues and/or People
- D8CORE-1291

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
